### DISCUSSION TITLE
Any plans for Monterey support?

### DISCUSSION THREAD

**Discussion by twilek** [The-Wineskin-Project/WineskinServer#266] [Q&A] [answered]
Hi,

I just updated my computer from Big Sur to Monterey since I thought most of the apps on my computer now have support for Monterey. But then I was going to use WineSkin and noticed that it now crashes when I try to do test runs and looked on this page and noticed that it only has support up to MacOSX 11. The image still runs, but WineSkin crashes in the background. Most of the functions in WineSkin seems to work though, the only thing I have noticed crashing so far is as I mentioned when I press "Test Run", not always though, but on most of the images that I have tested (that I know worked when running them in Big Sur). So my question now is: do you have any plans to upgrade WineSkin to fully work with Monterey in the near future?

Best Regards
Martin

**Comment by emendelson** [comment, marked as answer]
Wineskin works perfectly under Monterey on my system. I have five wrappers that I use almost every day. I've updated some older wrappers (see other discussions here) and created new ones, and some I continue using without change. Keep in mind that the file structure has changed in the latest wrapper (again, see other discussions for details).

**Reply by twilek** [reply]
Yes, as I mentioned, the wrappers still work, but if I open the Wineskin.app inside the wrapper and then go to Advanced and then press "Test Run", WineSkin often crashes. This function is only interesting if you create your own wrappers, so most people probably don't care about this, but since I was going to debug a wrapper I noticed this. And it didn't behave like this yesterday, before I upgraded to Monterey ;)

**Reply by twilek** [reply]
I just noticed that it doesn't seems to be a very big deal though, since the debug log is still written to disk so I am still able to get what I am looking for.

**Comment by emendelson** [comment]
That's very strange. I've just tried Test Runs in four different wrappers, and they all worked perfectly. Possibly the programs I'm running are less complicated than yours?

**Reply by twilek** [reply]
I just noticed another interesting thing, after I installed some wintricks on a wrapper, Wineskin stopped crashing when pressing "Test Run". Maybe this has something to do with some wrappers that was created under Big Sur that might cause this crash. Because after the wrapper was refreshed the crash seems to have disappeared for that specific wrapper.

Anyway, if Monterey is meant to be supported, I guess it could be a good idea to update the Readme.md on the GitHub front page?

**Comment by Gcenx** [comment]
macOS Monterey is not listed as supported as I’d not personally confirmed it was working though I didn’t see any reason _why_ it wouldn’t function.

Prior I’d confirm with other users, after the amount of issues that were opened that ignored the issue template and gave abuse when I’d requested the required information I’d given up and closed issues entirely.

As multiple users have confirmed no problems I can sadly update the readme to reflect this.

**Reply by twilek** [reply]
Ok, I see. As I mentioned above the issue I had seems to go away if I refresh a wrapper (yes, just noticed it is enough to just press "Refresh Wrapper"), so maybe it is related in some way to wrappers built on Big Sur and then run on Monterey. So maybe not even worth to troubleshoot after all. I just initially thought that Wineskin wasn't supported at all on Monterey so that was why I asked, but if this is the only problem I guess we can just forget about it :)

But if you despite this want to troubleshoot this I actually get an error dump if I start Wineskin via Porting Kit and can send over the dump to you if you want?

**Reply by emendelson** [reply]
> As multiple users have confirmed no problems I can sadly update the readme to reflect this.

I *hope* you mean "gladly"...

**Reply by Gcenx** [reply]
> I _hope_ you mean "gladly"...

I blame the lack of sleep
